plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+				'blockName'  => 'woocommerce/product-catalog-visibility-field',
+				'attributes' => [
+					'label'      => __( 'Hide in product catalog', 'woocommerce' ),
+					'visibility' => 'search',
+				],
+			]
+		);
+		$product_catalog_section->add_block(
+			[
+				'id'         => 'product-catalog-catalog-visibility',
+				'blockName'  => 'woocommerce/product-catalog-visibility-field',
+				'attributes' => [
+					'label'      => __( 'Hide from search results', 'woocommerce' ),
+					'visibility' => 'catalog',
+				],
+			]
+		);
+		$product_catalog_section->add_block(
+			[
+				'id'         => 'product-enable-product-reviews',
+				'blockName'  => 'woocommerce/product-checkbox-field',
+				'attributes' => [
+					'label'    => __( 'Enable product reviews', 'woocommerce' ),
+					'property' => 'reviews_allowed',
+				],
+			]
+		);
+		$product_catalog_section->add_block(
+			[
+				'id'         => 'product-post-password',
+				'blockName'  => 'woocommerce/product-password-field',
+				'attributes' => [
+					'label' => __( 'Require a password', 'woocommerce' ),
+				],
+			]
+		);
+		// Attributes section.
+		$product_catalog_section = $organization_group->add_section(
+			[
+				'id'         => 'product-attributes-section',
+				'attributes' => [
+					'title' => __( 'Attributes', 'woocommerce' ),
+				],
+			]
+		);
+		$product_catalog_section->add_block(
+			[
+				'id'        => 'product-attributes',
+				'blockName' => 'woocommerce/product-attributes-field',
+			]
+		);
+	}
+
+	/**
+	 * Adds the pricing group blocks to the template.
+	 */
+	private function add_pricing_group_blocks() {
+		$pricing_group = $this->add_group(
+			[
+				'id'         => 'pricing',
+				'order'      => 20,
+				'attributes' => [
+					'id'    => 'pricing',
+					'title' => __( 'Pricing', 'woocommerce' ),
+				],
+			]
+		);
+		$pricing_group->add_block(
+			[
+				'id'         => 'product_variation_notice_pricing_tab',
+				'blockName'  => 'woocommerce/product-has-variations-notice',
+				'attributes' => [
+					'id'         => 'wc-product-notice-has-options',
+					'content'    => __( 'This product has options, such as size or color. You can now manage each variation\'s price and other details individually.', 'woocommerce' ),
+					'buttonText' => __( 'Go to Variations', 'woocommerce' ),
+					'type'       => 'info',
+				],
+			]
+		);
+		// Product Pricing Section.
+		$product_pricing_section = $pricing_group->add_section(
+			[
+				'id'         => 'product-pricing-section',
+				'attributes' => [
+					'description' => sprintf(
+					/* translators: %1$s: Images guide link opening tag. %2$s: Images guide link closing tag.*/
+						__( 'Set a competitive price, put the product on sale, and manage tax calculations. %1$sHow to price your product?%2$s', 'woocommerce' ),
+						'<a href="https://woocommerce.com/posts/how-to-price-products-strategies-expert-tips/" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
```

Comment:
Any reason why `sprintf` was used instead of `createInterpolateElement`?

Reply:
Same reason as above: https://github.com/woocommerce/woocommerce/pull/39814#discussion_r1302177468